lib/ruby-debug-ide/attach/util.rb

```diff
@@ -1,5 +1,16 @@
 require 'ruby-debug-ide/attach/lldb'
 require 'ruby-debug-ide/attach/gdb'
+require 'socket'
+
+def get_child_pids(pid, pids = Array.new)
+  pids << pid
+  pipe = IO.popen("pgrep -P #{pid}")
```

Comment:
AFAIU `pgrep` is not available for MacOS; please try to make this solution 100% portable across nix systems